docker/docker-compose.yaml

```diff
@@ -2,24 +2,52 @@ version: '3.7'
 
 volumes:
   textile-fc-data:
+  textile-fc-ipfs:
+  textile-fc-lotus:
 
 services:
-  textile-fc:
-    network_mode: host
+  
+  fil-tools:
     build:
       context: ../
       dockerfile: exe/server/Dockerfile
-    environment:
-      - TEXFILTOOLS_LOTUSTOKEN=${TEXFILTOOLS_LOTUSTOKEN}
     ports:
+      - 8889:8889
       - 8888:8888
       - 6060:6060
-    restart: always
+      - 5002:5002
+      - 6002:6002
+    depends_on:
+      - ipfs
+      - lotus
+    environment:
+      - TEXFILTOOLS_LOTUSHOST=/dns4/lotus/tcp/1234
+      - TEXFILTOOLS_IPFSAPIADDR=/dns4/ipfs/tcp/5001
+      - TEXFILTOOLS_LOTUSTOKENFILE=/root/lotus/.lotus/token
+      - TEXFILTOOLS_REPOPATH=/root/fil-tools/.texfc
+    restart: unless-stopped
+    volumes:
+      - textile-fc-data:/root/fil-tools
+      - textile-fc-lotus:/root/lotus
```

Comment:
Leveraging docker-volumes to get the API-token from the _lotus_ container in the _fil-tools_ container.